tests/integration/message_batches/create_message_batches/test_field_validation.py

```diff
@@ -497,33 +415,187 @@ def test_null_personalisation(bearer_token_int, correlation_id, personalisation)
     """
     .. include:: ../../partials/validation/test_invalid_personalisation.rst
     """
-    resp = requests.post(f"{constants.INT_URL}{MESSAGE_BATCHES_ENDPOINT}", headers={
-        **headers,
-        "X-Correlation-Id": correlation_id,
-        "Authorization": bearer_token_int.value
-    }, json={
-        "data": {
-            "type": "MessageBatch",
-            "attributes": {
-                "routingPlanId": "b838b13c-f98c-4def-93f0-515d4e4f4ee1",
-                "messageBatchReference": str(uuid.uuid1()),
-                "messages": [
-                    {
-                        "messageReference": "72f2fa29-1570-47b7-9a67-63dc4b28fc1b",
-                        "recipient": {
-                            "nhsNumber": "9990548609",
-                            "dateOfBirth": "2023-01-01"
-                        },
-                        "personalisation": personalisation
-                    }
-                ]
-            }
-        }
-    })
+    data = Generators.generate_valid_create_message_batch_body("int")
+    data["data"]["attributes"]["messages"][0]["personalisation"] = personalisation
+    resp = requests.post(
+        f"{constants.INT_URL}{MESSAGE_BATCHES_ENDPOINT}",
+        headers={
+            **headers,
+            "X-Correlation-Id": correlation_id,
+            "Authorization": bearer_token_int.value
+        },
+        json=data
+    )
 
     Assertions.assert_error_with_optional_correlation_id(
         resp,
         400,
         Generators.generate_null_value_error("/data/attributes/messages/0/personalisation"),
         correlation_id
     )
+
+
+@pytest.mark.inttest
+@pytest.mark.parametrize("correlation_id", constants.CORRELATION_ID)
+def test_invalid_sms_contact_details(bearer_token_int, correlation_id):
+    """
+    .. include:: ../../partials/validation/test_invalid_contact_details_sms.rst
+    """
+    data = Generators.generate_valid_create_message_batch_body("int")
+    data["data"]["attributes"]["messages"][0]["recipient"]["contactDetails"] = {"sms": "077009000021"}
+    resp = requests.post(
+        f"{constants.INT_URL}{MESSAGE_BATCHES_ENDPOINT}",
+        headers={
+            "Authorization": bearer_token_int.value,
+            **headers,
+            "X-Correlation-Id": correlation_id
+        },
+        json=data
+    )
+
+    Assertions.assert_error_with_optional_correlation_id(
+        resp,
+        400,
+        Generators.generate_invalid_value_error_custom_detail(
+            "/data/attributes/messages/recipient/contactDetails/sms",
```

Comment:
I agree with you but I'm not sure what I can do to effectively manage this. Our architecture doesn't seem to allow us any way to say "run these tests against this version and run those tests against that version". What would you suggest?

Reply:
Since both the BE and the FE will be released at the same time, I personally think this PR should be updated to the latest specs. Then when it comes to the release we should deploy BE first (which will include this change) and then the FE.

We won't be able to fully test that it will work in int and prod but I don't think there's an easy way around it. If we don't accommodate for the message index now though, the release will fail and someone will have to make those changes in a hurry.

Happy to discuss if someone has a better idea :)

Reply:
I think the cleanest way to approach this would be to delay this PR to the next release but that might be overkill.

Reply:
Shouldn't these tests be failing against internal-dev now? 6415 is Done so should be deployed there.
I don't think we should be delaying things. I've not enjoyed it this sprint.

Reply:
Yes internal-dev should failed I think? I don't get it, just trying to understand :D

Reply:
Well we updated it to match the schema that dev is giving back _right now_, while the integration and production tests match what and integration and production return _right now_. I know that we need to update these for the release, but I don't see what else we can do right now if we want the nightly tests to continue to pass. We could agree that we expect the nightly tests to fail until the release, it's up to the team I suppose.

Arguably I would want the nightlies to, by default, run against the release tag of `v4.26.0` or whatever, rather than against the `release` branch, but we'll have to have that conversation separately. This is already holding up several other things.

Reply:
Ignore me, I wasn't looking at the latest changes... The internal-dev tests on this branch are expecting a message index so they are passing as expected.

Reply:
We reached a resolution for now, thankyou Simon